### PR TITLE
Poll pagination, bug fixes, package upgrades, e2e updates

### PR DESCRIPTION
**Pull request by hernandoagf**
Adds poll pagination, package upgrades, e2e updates, misc bug patches

### CODE REVIEW

File: modules/app/components/ResourceBox.tsx [hunk 1]
```diff
@@ -72,23 +75,23 @@ const resources: Record<ResourceType, Resource> = {
...
-        url: 'https://makerdao.world/en/learn/governance/how-voting-works#governance-polls-and-executive-votes'
+        url: 'https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance#how-is-the-voting-calculated'
       },
       {
-        linkTitle: 'How to set up your wallet for voting?',
-        url: 'https://makerdao.world/en/learn/governance/voting-setup/'
+        linkTitle: 'How to vote in on-chain governance?',
+        url: 'https://manual.makerdao.com/governance/voting-in-makerdao/how-to-vote'
       }
     ]
```
Comment (adamgoth) [inline modules/app/components/ResourceBox.tsx:96]: I think this is a duplicate (line 64 above has the same title/url)

File: modules/app/components/ResourceBox.tsx [hunk 2]
```diff
@@ -72,23 +75,23 @@ const resources: Record<ResourceType, Resource> = {
...
       {
         linkTitle: 'How to manually verify Executive Spells?',
-        url: 'https://makerdao.world/en/learn/governance/audit-exec-spells'
+        url: 'https://manual.makerdao.com/governance/verification/executive-audit'
       },
       {
         linkTitle: 'How is voting weight calculated?',
-        url: 'https://makerdao.world/en/learn/governance/how-voting-works#governance-polls-and-executive-votes'
+        url: 'https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance#how-is-the-voting-calculated'
       },
```
Comment (adamgoth) [inline modules/app/components/ResourceBox.tsx:91]: This one looks duplicated as well

Comment (hernandoagf) [reply modules/app/components/ResourceBox.tsx:91]: One of the links goes in the polling page and the other in the executive page, should we create another object for common links?

File: modules/polling/api/__tests__/sortPollList.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  for (let i = pollArray.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    const temp = pollArray[i];
+    pollArray[i] = pollArray[j];
+    pollArray[j] = temp;
+  }
+  return pollArray;
+};
+
+describe('Sort polls', () => {
```
Comment (adamgoth) [inline modules/polling/api/__tests__/sortPollList.ts:26]: should add `.spec` to this file name to not get it confused with actual code files

File: pages/api/polling/v2/all-polls.ts [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+import { PollsPaginatedResponse } from 'modules/polling/types/pollsResponse';
+
+/**
+ * @swagger
+ * paths:
+ *  /api/polling/v2/all-polls:
+ *    get:
+ *      tags:
+ *        - "polls"
+ *      description: Returns a paginated list of polls according to the specified filters and sorting.
```
Comment (adamgoth) [inline pages/api/polling/v2/all-polls.ts:24]: same here

**Review by adamgoth — DISMISSED** [review]

File: modules/app/components/ResourceBox.tsx (hunk 2 above)
Comment (adamgoth) [reply modules/app/components/ResourceBox.tsx:91]: I see now. We could but not a big deal, I think it's fine as is.

**Review by adamgoth — APPROVED** [review]

**Pull request merged**